tianshou/policy/modelfree/ddpg.py

```diff
@@ -37,20 +37,21 @@ class DDPGPolicy(BasePolicy):
         explanation.
     """
 
-    def __init__(self,
-                 actor: torch.nn.Module,
-                 actor_optim: torch.optim.Optimizer,
-                 critic: torch.nn.Module,
-                 critic_optim: torch.optim.Optimizer,
-                 tau: float = 0.005,
-                 gamma: float = 0.99,
-                 exploration_noise: Optional[BaseNoise]
-                 = GaussianNoise(sigma=0.1),
-                 action_range: Optional[Tuple[float, float]] = None,
-                 reward_normalization: bool = False,
-                 ignore_done: bool = False,
-                 estimation_step: int = 1,
-                 **kwargs) -> None:
+    def __init__(
+        self,
+        actor: Optional[torch.nn.Module],
+        actor_optim: Optional[torch.optim.Optimizer],
+        critic: Optional[torch.nn.Module],
+        critic_optim: Optional[torch.optim.Optimizer],
+        tau: float = 0.005,
+        gamma: float = 0.99,
+        exploration_noise: Optional[BaseNoise] = GaussianNoise(sigma=0.1),
+        action_range: Optional[Tuple[float, float]] = None,
```

Comment:
should not be None

Reply:
Why not ? None could be replaced by [-np.inf, np.inf] automatically. I think it makes sense.